numpy/core/tests/test_numeric.py

```diff
@@ -2953,7 +2953,7 @@ def test_sparse(self):
         assert_array_equal(x, np.array([[0], [1], [2], [3]]))
         assert_array_equal(y, np.array([[0, 1, 2]]))
 
-    @pytest.mark.parametrize("dtype", [np.int, np.float32, np.float64])
+    @pytest.mark.parametrize("dtype", [int, np.float32, np.float64])
```

Comment:
maybe `np.int64` to parallel the other dtypes? It seems a bit odd to use a builtin here